session.go

```diff
@@ -1248,6 +1260,14 @@ func (q *Query) reset() {
 	*q = Query{}
 }
 
+var (
+	lwtRegexp = regexp.MustCompile("(.*IF( *)(NOT)?( *)EXISTS$)|((.*)IF( *)(.*)=(.*))")
+)
+
+func (q *Query) lwtQuery() {
```

Comment:
I'd suggest implementing it as `func isLWT(stmt string) bool` much easier to test

Reply:
The messy bits are in the shuffle tests where replicas has to be set up.

Reply:
Not relevant after using the metadata bit.